tools/fast/action_miner.go

```diff
@@ -65,17 +67,27 @@ func (f *Filecoin) MinerOwner(ctx context.Context, minerAddr address.Address) (a
 	return out, nil
 }
 
-// MinerPower runs the `miner power` command against the filecoin process
-func (f *Filecoin) MinerPower(ctx context.Context, minerAddr address.Address) (*big.Int, error) {
-	var out big.Int
+// MinerPower runs the `miner power` command against the filecoin process and returns the result
+// as two *big.Ints
+func (f *Filecoin) MinerPower(ctx context.Context, minerAddr address.Address) (int64, int64, error) {
```

Comment:
I know this is technically out of line with power's type, but when writing the test for this, it seemed silly to output big.Ints;  in the test we'd be creating big.Ints from uint64s, we'd be parsing to (u)ints from strings, and converting that to big.Int, so .... skipping the middleman for testing purposes seemed reasonable.

Reply:
Ok for now, but note that the comment above is wrong.

FAST is not just for testing, so this is a shortcut that will trip someone up later.